### DISCUSSION TITLE
Manual/UIE-supported-tags

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#521] [Notes]
# Manual/UIE-supported-tags

https://docs.unity3d.com/Manual/UIE-supported-tags.html

**Comment by vertxxyz** [comment]
If you want to indent every subsequent line in a paragraph, I have used an `<line-indent=-15px>...</line-indent>` tag around a portion of text in the first line, and added a `margin-left: 15px;` to the USS for that element to counter the outdent.
Other setups might indent the first line too, or they won't indent lines that are word-wrapped.
![image](https://user-images.githubusercontent.com/21963717/212595439-d73caebd-eb34-4068-a974-b219e0aabba6.png)

**Comment by vertxxyz** [comment]
The font tag will display as is (in the clear) if there is no [Font Asset](https://docs.unity3d.com/Manual/UIE-font-asset-landing.html) in the Resources folder that matches the name.
